packages/react-i18n-universal-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/react-i18n-universal-provider",
-  "version": "2.1.21",
+  "version": "2.1.22-currency.0",
```

Comment:
I'm not sure about the version control here. Why do we need it to be xx-curency.0?

Reply:
Yes, I made a beta release yesterday to test it, I'm ok with moving it to `2.1.22`